elasticdl/python/master/task_manager.py

```diff
@@ -30,6 +29,7 @@
 from elasticdl.python.data.reader.data_reader_factory import create_data_reader
 
 _MAX_TASK_RETRIES = 3
+_TASK_TIMEOUT_THRESHOLD = 300
```

Comment:
_TASK_TIMEOUT_THRESHOLD -> _TASK_TIMEOUT_THRESHOLD_MS, we can tell the reader that the unit of the time is milliseconds.

Reply:
Good suggestion.